docs/developer-guide/integrations/users/openId.md

```diff
@@ -0,0 +1,168 @@
+# Integration with OpenID connect
+
+MapStore allows to integrate and login using some common [OpenID connect](https://openid.net/connect/) services. With these configurations, you can add to the login form some buttons to login with the given service.
```

Comment:
I would say as follow:

"MapStore provides an integration with [OpenID connect](https://openid.net/connect/) services. Having this support properly configured, you can make MapStore users able to login with the given OpenID service.
 "